MERN/Topic/Backend/Node/4.2_Example of using modules.md

```diff
@@ -0,0 +1,39 @@
+   #    IMPORT A MODULE
+- In your project directory, initialise npm package by the command below,to create the package.json file.
+
+  ``` 
+  npm init 
+  ```    
+  
+- Now ,install the module  `http` .  
+  
+  ``` 
+  npm install http 
+  ```  
+
+  **You can view the module with version in package-lock.json file**
+  ![Screenshot (287)](https://user-images.githubusercontent.com/68144616/134821166-d3a61c42-5d62-4131-ac92-6da7359685ec.png)
+
+- Now in your index.js file
+  Import the module by using the code below.
+
+  ```
+  var http= require(“http”)
+  ```  
+  
+ 
+- Now test its working
+
+  ```
+  
+  var http = require('http');
+
+  var server = http.createServer(function(req, res){
+
+  console.log(“server created”);
+
+  });
+  server.listen(5000);
```

Comment:
It would be better if you can add a callback function to the listener which logs something like 'Server is running on PORT 5000' just to make the user's know that that the server is created and running.